Strings/strings_home_task.py

```diff
@@ -0,0 +1,55 @@
+"""
+Home task for Module 3 - Strings
+
+"""
+
+# Creation of variable as required at home task
+corrupted_text = """homEwork:
+
+  tHis iz your homeWork, copy these Text to variable.
+
+
+
+  You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+
+
+  it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE.
+
+
+
+  last iz TO calculate nuMber OF Whitespace characteRS in this Text. caREFULL, not only Spaces, but ALL whitespaces. I got 87."""
+
+# Cleaning of text and adding it to the one list
+formatted_text = []  # Creatre an empty list for formatted text
+for paragraph in corrupted_text.split('.\n\n\n\n  '):  # Split paragraphs by \n\n\n\n
+    temporary_list_for_paragraph = []  # Initialize separate list for one paragraph for further operations
+    temporary_string_for_paragraph = ''  # Initialize temporary string for one paragraph
```

Comment:
Do you really need it?
```suggestion
    temporary_list_for_paragraph = []  # Initialize separate list for one paragraph for further operations
    temporary_string_for_paragraph = ''  # Initialize temporary string for one paragraph
```

Reply:
I'll save it for now and try to avoid using temp lists and strings in functions home task.